Sources/BraintreeCore/BTAppContextSwitcher.swift

```diff
@@ -13,7 +13,10 @@ import UIKit
     /// The URL scheme to return to this app after switching to another app or opening a SFSafariViewController.
     /// This URL scheme must be registered as a URL Type in the app's info.plist, and it must start with the app's bundle ID.
     public var returnURLScheme: String = ""
-    
+
+    /// The URL to use for the PayPal universal link flow. Must contain the path `braintree-payments`.
+    public var universalLink: String = ""
```

Comment:
Do we want a CHANGELOG entry for this?

Reply:
Yes!

Reply:
Done: https://github.com/braintree/braintree_ios/pull/1214/commits/69332b848acb3ba53e5fb519c9192ada6e9e67be